dotnet/samples/KernelSyntaxExamples/Example43_GetModelResult.cs

```diff
@@ -35,13 +35,13 @@ public static async Task RunAsync()
         var textResult = await myFunction.InvokeAsync("Sci-fi",
             settings: new CompleteRequestSettings { ResultsPerPrompt = 3, MaxTokens = 500, Temperature = 1, TopP = 0.5 });
         Console.WriteLine(textResult);
-        Console.WriteLine(textResult.ModelResults.Select(result => result.GetOpenAITextResult()).AsJson());
+        Console.WriteLine(textResult.ModelResults.Select(result => result.GetOpenAIChatResult()).AsJson());
```

Comment:
There was one PR that updated all models in examples to chat completion. Since that change was pushed, this example is not working. This PR should fix this particular example.

Reply:
Yeah, we should really make that better. If I call a semantic function from a native function I do not always know what AI configuration has been made with the kernel so how can I best do this kind of action there.